### ISSUE TITLE
PAGINATION_DIRECTIVES  doesn't work on demo.

### ISSUE THREAD

**Issue by gtvforever** [valor-software/ng2-table#273]
Hi, just copied the demo ts & html into my project.
When i tried to launch the app with the command 'npm start'. i got below error message:
'(20,9): error TS2305: Module '"c:/Users/ngtable/node_modules/ng2-bootstrap/ng2-bootstrap"' has no exported member 'PAGINATION_DIRECTIVES'. '

i already installed the ng-bootstrap 1.1.0.

Is there anything i missed?

Thank you in advance.

**Comment by otelnov** [comment]
https://github.com/valor-software/ng2-table/pull/268
ng2-tables will be update soon to work with new ng2-bootstrap version

**Comment by gtvforever** [comment]
Thank you otelnov.  Just want to double confirm that your fixing have fixed the  issue i met, right?

**Comment by RomanVolkov** [comment]
Yes. I installed "ng2-bootstrap": "1.1.0" too and using angular 2.0.0-rc5. Doesn't work too

**Comment by ghost** [comment]
Do you need to execute 

<pre>
npm i ng2-bootstrap --save
</pre>

after this include  in your module:

<pre>
import {PaginationComponent, PaginationConfig} from 'ng2-bootstrap/ng2-bootstrap';
</pre>

And incluede in your app.module

<pre>
import {PaginationComponent, PaginationConfig} from 'ng2-bootstrap/ng2-bootstrap';

  imports: [
        PaginationComponent
    ],
providers: [
        PaginationConfig
    ]
</pre>
